lib/app/service/storage_service/nekoton_repository_service.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:app/app/service/service.dart';
+import 'package:injectable/injectable.dart';
+import 'package:nekoton_repository/nekoton_repository.dart';
+
+/// This is a wrapper-class above [NekotonRepository] that provides methods
+/// to interact with NekotonRepository.
+@singleton
+class NekotonRepositoryService extends AbstractStorageService {
```

Comment:
Imho it would be better `NekotonRepositoryStorageService `

Reply:
🤟